### ISSUE TITLE
Checkbox A11y Issues

### ISSUE THREAD

**Issue by CillianHPE** [grommet/hpe-design-system#4924]
### Link to problem

https://csvjson.com/csv2json

### Current problem

Checkbox compoennt fials the following WCAG rules

- 1.3.6  Screen reader reads out misleading information
- 2.5.5 Checksbox does not meet the AAA 44px X 44px minimum touch requirements

**Comment by CillianHPE** [comment]
Closes as this issue was created in the wrong repo.

Moved to Grommet repo.

https://github.com/grommet/grommet/issues/7558
